docs/utilities/storage.mdx

```diff
@@ -140,6 +179,8 @@ const content = {} // anything JSON serializable
 const storedAtom = atomWithStorage('stored-key', content, storage)
 ```
 
+**Note** you probably want to set `getOnInit` to true if you're expecting the storage value to be available immediately instead of the default value
```

Comment:
This is RN section, so instead of adding another note here, how about changing the example above?